Tools/Scripts/webkitpy/benchmark_runner/run_benchmark.py

```diff
@@ -55,7 +55,8 @@ def config_argument_parser():
     parser.add_argument('--diagnose-directory', dest='diagnose_dir', default=diagnose_directory, help='Directory for storing diagnose information on test failure. Defaults to {}.'.format(diagnose_directory))
     parser.add_argument('--no-adjust-unit', dest='scale_unit', action='store_false', help="Don't convert to scientific notation.")
     parser.add_argument('--show-iteration-values', dest='show_iteration_values', action='store_true', help="Show the measured value for each iteration in addition to averages.")
-    parser.add_argument('--generate-profiles', dest='generate_profiles', action='store_true', help="Collect LLVM profiles for PGO, and copy them to the diagnose directory.")
+    parser.add_argument('--generate-pgo-profiles', '--generate-profiles', dest="generate_pgo_profiles", action='store_true', help="Collect LLVM profiles for PGO, and copy them to the diagnostics directory.")
```

Comment:
Don't we need to change the references to `--generate-profiles`  on `Tools/Scripts/build-and-collect-pgo-profiles`?

Reply:
We do, but that's for a separate patch. This is backwards compatible, so `--generate-profiles` will still collect PGO profiles (for now).

Reply:
I think they should be in one patch, otherwise, in the gap between this patch and that patch, invoking this script with `--generate-profiles` will fail.

Reply:
`--generate-profiles` will still work after this is landed though, I was planning on a separate patch to change `build-and-collect-pgo-profiles` to the new flag and remove the old one after this patch lands. Or unless you are saying it would be a better idea to do it all in one go?

Reply:
Ah, missed `'--generate-profiles'`